tests/define_test_aggregate_project.py

```diff
@@ -0,0 +1,77 @@
+from flow import FlowProject, aggregator, cmd, with_job
+
+
+class _TestAggregateProject(FlowProject):
+    pass
+
+
+def custom_aggregator(jobs):
```

Comment:
This name is too generic and will make it harder for readers of the tests to know what the custom aggregation is based on. Rename it to something like `sp_i_parity_aggregator`.

Reply:
On second thought, I guess `parity` may not be a word everyone's familiar with, I'm not sure. If the next reviewer who sees this thinks that might be the case, then maybe we use an even simpler name like `sp_i_even_odd_aggregator`. Either name is fine with me, although my first suggestion is a bit less verbose :)

Reply:
Resolved in 1f3efb6b4ff57c8e9094347e3bc9a5a71ca22251.